riff-raff/public/docs/riffraff/administration/auth.md

```diff
@@ -8,6 +8,19 @@ Authentication
 Authentication is handled by Google. This obtains the users full name and e-mail address and both of these are used to 
 provide an audit trail of deployments and other actions in the app.
 
+Google auth is implemented in two layers
+- Load Balancer layer: this component is defined in the main riff raff cloudformation template and uses [authenticate-oidc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#configure-user-authentication). The template relies on a secret stored in SSM.
```

Comment:
It might be worth adding a link here?
```suggestion
- Load Balancer layer: this component is defined in [the main riff raff cloudformation template](https://github.com/guardian/deploy-tools-platform/blob/main/cloudformation/riffraff/riffraff.template.yaml) and uses [authenticate-oidc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/listener-authenticate-users.html#configure-user-authentication). The template relies on a secret stored in SSM.
```

Reply:
Thanks! I avoided adding links because this is a public repo, but I checked and there's already mention of `deploy-tools-platform` in this repo so might as well.